.github/workflows/publish_s3.yml

```diff
@@ -189,7 +189,7 @@ jobs:
           aws-region: ${{ secrets.AWS_REGION }}
       - id: git_hash
         name: Get Git hash for "${{ needs.workflow_data.outputs.commit_id }}"
-        run: echo "hash=$(git log -1 --pretty=format:"%H" "${{ needs.workflow_data.outputs.commit_id }}")" | tee -a $GITHUB_OUTPUT
+        run: echo "hash=$(git log -1 --pretty=format:"%H" "${{ needs.workflow_data.outputs.commit_id }}")" -- . | tee -a $GITHUB_OUTPUT
```

Comment:
should this not be an argument to git? I think then it should be within the parentheses, if I'm not mistaking

Reply:
You're totally right.